Bugsnag/Client/BugsnagClient.m

```diff
@@ -546,6 +546,11 @@ - (void)start {
     [self.crashSentry install:self.configuration
                     apiClient:self.errorReportApiClient
                       onCrash:&BSSerializeDataCrashHandler];
+    // overridden elsewhere for handled errors, so we can assume that this only
+    // applies to unhandled errors
+    BSGThreadSendPolicy sendThreads = self.configuration.sendThreads;
+    [BSG_KSCrash sharedInstance].threadTracingEnabled = sendThreads == BSGThreadSendPolicyAlways
+                                                     || sendThreads == BSGThreadSendPolicyUnhandledOnly;
```

Comment:
As per previous comment, can this be done in `BSG_KSCrash.m` with all the other values that are overridden by our own configuration?